e2e/app/component/admin-table.ts

```diff
@@ -0,0 +1,62 @@
+import { Page } from 'puppeteer';
+import Container from 'app/container';
+//import { getPropValue } from 'utils/element-utils';
+import Table from './table';
+
+
+
+const defaultXpath = '//*[contains(concat(normalize-space(@class), " "), "p-datatable ")]';
+
+
+export default class adminTable extends Table {
```

Comment:
This should be named userAdminTable or similar.  There are other admin tables besides this one.

Reply:
@jmthibault79 Sorry! I missed your comment here. I've named this component as AdminTable, since i've included the xpath for both the user admin and institution admin table in this component file. Please let me know if you still want me to rename it. 
Also, this PR is now stale. I've a new PR for the same test case. https://github.com/all-of-us/workbench/pull/5152

Reply:
OK thanks.  if we're reusing this, the name makes sense.

Reply:
Please close this PR with a comment pointing to the new one.

Reply:
Thanks @jmthibault79 ! Closing this PR. 
A new [PR ](https://github.com/all-of-us/workbench/pull/5152) for user-admin-UI test case.